HyperV/scripts/generateCinderCfg.ps1

```diff
@@ -0,0 +1,64 @@
+Param(
+    [Parameter(Mandatory=$true)][string]$configDir,
+    [Parameter(Mandatory=$true)][string]$templatePath,
+    [Parameter(Mandatory=$true)][string]$serverIP,
+    [Parameter(Mandatory=$true)][string]$rabbitUser,
+    [Parameter(Mandatory=$true)][string]$logDir,
+    [Parameter(Mandatory=$true)][string]$lockPath
+)
+
+
+$username = 'Administrator'
+$password = 'H@rd24G3t'
+
+
+function unzip($src, $dest) {
```

Comment:
We could move this to utils.ps1, maybe we'll need to use it in other places as well.